bleak/backends/bluezdbus/client.py

```diff
@@ -176,6 +177,15 @@ def on_value_changed(char_path: str, value: bytes) -> None:
                                 member="Connect",
                             )
                         )
+                    if (
+                        reply is not None
+                        and reply.message_type == MessageType.ERROR
+                        and reply.error_name == ErrorType.UNKNOWN_OBJECT
```

Comment:
Are you sure about the UnknownObject error? The documentation does not mention it:
https://github.com/bluez/bluez/blob/master/doc/device-api.txt